app/lib/filters/show_all_applications.rb

```diff
@@ -0,0 +1,25 @@
+module Filters
+  class ShowAllApplications < Base
+    def apply
+      unless show_all_applications.include?("show_all")
+        ninety_days_ago = 90.days.ago
+        new_scope =
+          scope.where(
+            "(awarded_at >= :ninety_days OR awarded_at IS NULL)
+          AND (declined_at >= :ninety_days OR declined_at IS NULL)
+          AND (withdrawn_at >= :ninety_days OR withdrawn_at IS NULL)",
+            ninety_days: ninety_days_ago,
+          )
+        return new_scope
```

Comment:
```suggestion
        return scope.where(
            "(awarded_at >= :ninety_days OR awarded_at IS NULL)
          AND (declined_at >= :ninety_days OR declined_at IS NULL)
          AND (withdrawn_at >= :ninety_days OR withdrawn_at IS NULL)",
            ninety_days: ninety_days_ago,
          )
```

I don't think we need to define a variable here.